resources/quality/ultimaker_methodxl/um_methodxl_labs_um-pla-175_0.2mm.inst.cfg

```diff
@@ -20,34 +20,26 @@ cool_fan_speed_0 = 0
 cool_min_layer_time = 10
 cool_min_speed = 5
 cool_min_temperature = 205
-default_material_bed_temperature = 45
 infill_sparse_density = 15
-material_bed_temperature = =default_material_bed_temperature
-material_bed_temperature_layer_0 = =default_material_bed_temperature + 5
-material_final_print_temperature = 210
-material_initial_print_temperature = 210
-material_print_temperature = 210
+material_final_print_temperature = 205
+material_initial_print_temperature = 205
 material_print_temperature_layer_0 = 225
-material_standby_temperature = 160
 min_bead_width = =line_width * 0.75
 min_wall_line_width = =line_width * 0.75
 raft_airgap = 0.18
-raft_base_line_spacing = 4
-raft_base_line_width = 1.2
-raft_base_speed = =raft_speed
+raft_base_line_spacing = =4 if extruder_nr == raft_interface_extruder_nr else 2.8
+raft_base_line_width = =1.2 if extruder_nr == raft_interface_extruder_nr else 1.4
+raft_base_speed = 15
```

Comment:
I think that this should be the old one (so "raft_base_speed = =raft_speed"), as otherwise if the user changes the raft_speed, the base isn't changing alongside with it.